Lombiq.NodeJs.Extensions/Docs/Markdown.md

```diff
@@ -1,14 +1,26 @@
 ﻿# Scripts for MD files
 
+## Usage
+
 Since Markdown files don't need to be built, linting isn't included in the normal workflow but a separate target can launch it for any project that uses _Lombiq.NodeJs.Extensions_. You can configure its behavior with the following MSBuild property:
 
 - `<MarkdownAnalysis>`:
   - If set to "false" or if unset (default), Markdown linting is disabled.
   - If set to "true", it lints every _md_ file inside the project's directory.
-  - If set to "solution", it lints every _md_ file inside the `$(SolutionDir)`. This is useful to catch files not covered by an individual project, such as the root Readme. However, it can cause duplicate warnings for files in another project that uses `Lombiq.NodeJs.Extensions`.
+  - If set to "solution", it lints every _md_ file inside the `$(SolutionDir)`. This is useful to catch files not covered by an individual project, such as the root Readme. However, it can cause duplicate warnings for files in another project that uses `Lombiq.NodeJs.Extensions`. We suggest setting this value in the solution's "entry" project such your Web project (e.g. [Lombiq.OSOCE.NuGet.Web.csproj](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/blob/dev/NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj)). If you are not doing anything else there, also include the properties `<ExecDotnetPostcleanCommand>false</ExecDotnetPostcleanCommand>` and `<ExecDotnetPrebuildCommand>false</ExecDotnetPrebuildCommand>` to avoid running the JS/SCSS/Assets pipelines.
+
+> ℹ If you include the repository as submodule and want Solution-wide Markdown analysis it's better to add the [Lombiq.NodeJs.Extensions.SolutionMarkdownAnalysis](../../Lombiq.NodeJs.Extensions.SolutionMarkdownAnalysis/Readme.md) project to your solution.
```

Comment:
Do we actually need to NuGet-publish `SolutionMarkdownAnalysis`?

Reply:
No. I even removed it from the _Lombiq.NodeJs.Extensions.sln_.

Reply:
Because it has all the NuGet bells and whistles in its folder and csproj, and the NuGet shield in the Readme.

Reply:
Ok, removed those too.